mix.exs

```diff
@@ -1,15 +1,24 @@
 defmodule ElixirTools.MixProject do
   use Mix.Project
 
+  @version "0.1.0"
```

Comment:
What does this have to do with status badge?

Reply:
rebased